web/js/projectDetails.js

```diff
@@ -80,22 +33,46 @@ var projectComboBox = new Ext.form.ComboBox({
         autoLoad: true,  //initial data are loaded in the application init
         autoSave: false, //if set true, changes will be sent instantly
         baseParams: {
-            'cid': customerId,
             'order': 'description',
             'active': 'false',
         },
-        proxy: new Ext.data.HttpProxy({url: 'services/getCustomerProjectsService.php', method: 'GET'}),
+        filter: function(property, value, anyMatch, caseSensitive) {
+            var fn;
+            if (((property == 'description') || (property == 'customerName')) && !Ext.isEmpty(value, false)) {
+                value = this.data.createValueMatcher(value, anyMatch, caseSensitive);
+                fn = function(r){
+                    return value.test(r.data['description']) || value.test(r.data['customerName']);
+                };
+            } else {
+                fn = this.createFilterFn(property, value, anyMatch, caseSensitive);
+            }
+            return fn ? this.filterBy(fn) : this.clearFilter();
+        },
+        proxy: new Ext.data.HttpProxy({url: 'services/getProjectsAndCustomersForLoginService.php', method: 'GET'}),
         reader:new Ext.data.XmlReader({record: 'project', id:'id' }, projectRecord),
         remoteSort: false,
     }),
     mode: 'local',
     valueField: 'id',
     typeAhead: true,
```

Comment:
This is causing some trouble when you type the name of the customer, because it will replace what you typed with the project name of the first entry. It should work just fine when this value is false.